waypoints-special/CAN_PG_A-M.cup

```diff
@@ -1,415 +1,414 @@
+ObsZone=3,Style=3,R1=2000m,A1=180,Move=1
+"LSC Triangle 2","Cowley a/p","Cowley a/p","Chokio","Twin Butte","Cowley a/p","Cowley a/p",,,,
 Options,WpDis=False,NearDis=0.0km,NearAlt=0.0ft
-ObsZone=0,Style=2,R1=2000m,A1=180
+ObsZone=0,Style=2,R1=2000m,A1=180,Move=1
 ObsZone=1,Style=1,R1=2000m,A1=180
 ObsZone=2,Style=1,R1=2000m,A1=180
-ObsZone=3,Style=3,R1=2000m,A1=180
-"LSC Triangle 2 (Rev)","Cowley a/p","Cowley a/p","Twin Butte","Chokio","Cowley a/p","Cowley a/p"
+ObsZone=3,Style=3,R1=2000m,A1=180,Move=1
+"LSC Triangle 2 (Rev)","Cowley a/p","Cowley a/p","Twin Butte","Chokio","Cowley a/p","Cowley a/p",,,,
 Options,WpDis=False,NearDis=0.0km,NearAlt=0.0ft
-ObsZone=0,Style=2,R1=2000m,A1=180
+ObsZone=0,Style=2,R1=2000m,A1=180,Move=1
 ObsZone=1,Style=1,R1=2000m,A1=180
 ObsZone=2,Style=1,R1=2000m,A1=180
-ObsZone=3,Style=3,R1=2000m,A1=180
-"LSC Triangle 3","Cowley a/p","Cowley a/p","Parkview","Standoff","Cowley a/p","Cowley a/p"
+ObsZone=3,Style=3,R1=2000m,A1=180,Move=1
+"LSC Triangle 3","Cowley a/p","Cowley a/p","Parkview","Standoff","Cowley a/p","Cowley a/p",,,,
 Options,WpDis=False,NearDis=0.0km,NearAlt=0.0ft
-ObsZone=0,Style=2,R1=2000m,A1=180
+ObsZone=0,Style=2,R1=2000m,A1=180,Move=1
 ObsZone=1,Style=1,R1=2000m,A1=180
 ObsZone=2,Style=1,R1=2000m,A1=180
-ObsZone=3,Style=3,R1=2000m,A1=180
-"LSC Triangle 3 (Rev)","Cowley a/p","Cowley a/p","Standoff","Parkview","Cowley a/p","Cowley a/p"
+ObsZone=3,Style=3,R1=2000m,A1=180,Move=1
+"LSC Triangle 3 (Rev)","Cowley a/p","Cowley a/p","Standoff","Parkview","Cowley a/p","Cowley a/p",,,,
 Options,WpDis=False,NearDis=0.0km,NearAlt=0.0ft
-ObsZone=0,Style=2,R1=2000m,A1=180
+ObsZone=0,Style=2,R1=2000m,A1=180,Move=1
 ObsZone=1,Style=1,R1=2000m,A1=180
 ObsZone=2,Style=1,R1=2000m,A1=180
-ObsZone=3,Style=3,R1=2000m,A1=180
-"MSC Racetrack","Hawkesbury","Hawkesbury","Blue Corners","Vankleek Hill","Trans Canada","Hawkesbury","Hawkesbury"
+ObsZone=3,Style=3,R1=2000m,A1=180,Move=1
+"MSC Racetrack","Hawkesbury","Hawkesbury","Blue Corners","Vankleek Hill","Trans Canada","Hawkesbury","Hawkesbury",,,
 Options,WpDis=False,NearDis=0.0km,NearAlt=0.0ft
 ObsZone=0,Style=2,R1=2000m,A1=180,Move=1
 ObsZone=1,Style=1,R1=2000m,A1=180
 ObsZone=2,Style=1,R1=2000m,A1=180
 ObsZone=3,Style=1,R1=2000m,A1=180
 ObsZone=4,Style=3,R1=2000m,A1=180,Move=1
-"MSC Racetrack rev","Hawkesbury","Hawkesbury","Trans Canada","Vankleek Hill","Blue Corners","Hawkesbury","Hawkesbury"
+"MSC Racetrack rev","Hawkesbury","Hawkesbury","Trans Canada","Vankleek Hill","Blue Corners","Hawkesbury","Hawkesbury",,,
 Options,WpDis=False,NearDis=0.0km,NearAlt=0.0ft
 ObsZone=0,Style=2,R1=2000m,A1=180,Move=1
 ObsZone=1,Style=1,R1=2000m,A1=180
 ObsZone=2,Style=1,R1=2000m,A1=180
 ObsZone=3,Style=1,R1=2000m,A1=180
 ObsZone=4,Style=3,R1=2000m,A1=180,Move=1
-"MSC Sprint","Hawkesbury","Hawkesbury","Pendleton","Alexandria","Hawkesbury","Hawkesbury"
+"MSC Sprint","Hawkesbury","Hawkesbury","Pendleton","Alexandria","Hawkesbury","Hawkesbury",,,,
```

Comment:
Another instance of repeated waypoints. Please correct all of them.

Reply:
At least in my version of SeeYou, the first Hawkesbury is the takeoff point and the second is the start.

Reply:
I observed this too now on my version of seeyou.